lib/Elastica/Suggest/AbstractSuggest.php

```diff
@@ -1,12 +1,13 @@
 <?php
 namespace Elastica\Suggest;
 
+use Elastica\Nameable;
 use Elastica\Param;
 
 /**
  * Class AbstractSuggestion.
  */
-abstract class AbstractSuggest extends Param
+abstract class AbstractSuggest extends Param implements Nameable
```

Comment:
My commet above + `AbstractSuggest` should have setter for name regardless of new interface.